playground/frontend/build.gradle

```diff
@@ -40,26 +40,18 @@ dependencies {
   dockerDependency project(path: playgroundJobServerProject, configuration: "shadow")
 }
 
-task removeBuild  {
-  group = "verification"
-  description = "remove build artifacts"
-  doLast {
-    exec {
-      executable("rm")
-      args("-r", "build")
```

Comment:
`flutter clean` is added instead which additionally deletes `.dart_tool` directory. Somehow this is required for `flutter_gen_runner` package which itself looks more like a bug, but still.